pkg/abstractions/endpoint/task.go

```diff
@@ -30,7 +32,7 @@ func (t *EndpointTask) Execute(ctx context.Context, options ...interface{}) erro
 		return err
 	}
 
-	return instance.buffer.ForwardRequest(echoCtx, t)
+	return instance.buffer.ForwardRequest(echoCtx, t, payload)
```

Comment:
This is a little hacky, but it makes sure that endpoint request payloads don't get stored in redis along with the rest of the task message. Since endpoint tasks don't support retries, this isn't necessary and wastes redis memory.